### PR TITLE
added a fyyd verification module

### PR DESCRIPTION
**Pull request by eazyliving**
This module adds a verification token for the podcast search engine fyyd to your feeds.

### CODE REVIEW

**Comment by eteubert** [comment]
Looking great. Is there any way to generate a link to the fyyd podcast website? Or would that need additional information?

**Comment by eazyliving** [comment]
You think about additional options to integrate fyyd, don't you?
That also came to my mind yesterday and it will totally happen. 

Linking to fyyd happens with a numeric ID (atm). For example 'Freak Show' is number 85: https://fyyd.de/podcast/85/0

Next up: Why pubsubhubbbub? Why not ping fyyd directly? For new episodes or updated images (which is a nuisance all the time with any directory)

I'm sure, there are a lot of services you could include into this module :-)

**Comment by timpritlove** [comment]
One thing to add would be a link to the official fyyd logo in square format so that we can use this. Podlove Publisher should also provide a API to have fyyd registered internally as a directory item so that we can list all official directory instances of the podcast.

File: lib/modules/fyyd/fyyd.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+					return;
+			add_action( 'podlove_rss2_head', function( $feed ) use ( $fyyd_verifycode ) {
+					echo "\n\t" . sprintf( '<fyyd:verify xmlns:fyyd="https://fyyd.de/fyyd-ns/">%s</fyyd:verify>'."\n\t", $fyyd_verifycode );
+			} );
+
+	}
+
+}
+
+?>
```
Comment (eteubert) [inline lib/modules/fyyd/fyyd.php]: Do not end files with `?>`. The trailing newline leads to many issues, like newlines before feeds can be output by WordPress and thereby generating invalid feeds.

**Comment by eteubert** [comment]
was merged manually some time ago

**Pull request closed without merging**